### PR TITLE
Proposing an issue template for huboard-web

### PR DESCRIPTION
**Pull request by discorick**
Let tighten up our new issue creation! Looking for suggestions and feedback.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### Description:
+
+#### Steps to reproduce (if applicable):
+
+#### Helpful external resources (blog, docs etc.):
```
Comment (dahlbyk) [inline ISSUE_TEMPLATE.md]: Comma after docs :trollface:

**Comment by dahlbyk** [comment]
Do we want to give any guidance as to what might be opened here vs on huboard/huboard? (I honestly don't know what our guidance would be; I've mostly lived here.)

**Comment by discorick** [comment]
I think now is a good time to have that conversation - theirs a few ways we could go about doing it, like locking issues in one way or another. I think @rauhryan was thinking bugs should be opened on huboard-web and feature requests via huboard/huboard

**Comment by dahlbyk** [comment]
> I think @rauhryan was thinking bugs should be opened on huboard-web and feature requests via huboard/huboard

I'd be fine trying to funnel everything external through huboard/huboard, reserving repo-specific issues for bugs/debt that we identify. We might even make the template here simply a comment referring people to https://github.com/huboard/huboard-web/issues/new.

**Pull request merged**